test/Microsoft.IdentityModel.Tokens.Tests/Base64UrlEncodingTests.cs

```diff
@@ -307,5 +307,25 @@ public class Base64UrlEncoderTheoryData : TheoryDataBase
 
             public bool EncodingOnly { get; set; } = false;
         }
+
+        [Fact]
+        public void ValidateAndGetOutputSizeTests()
+        {
+            string input = string.Empty;
+            Assert.Throws<ArgumentNullException>(() => Base64UrlEncoding.ValidateAndGetOutputSize(input.AsSpan(), 0, 0));
+            Assert.Throws<ArgumentException>(() => Base64UrlEncoding.ValidateAndGetOutputSize("abc".AsSpan(), -1, 3));
+            Assert.Throws<ArgumentException>(() => Base64UrlEncoding.ValidateAndGetOutputSize("abc".AsSpan(), 0, -1));
+            Assert.Throws<ArgumentException>(() => Base64UrlEncoding.ValidateAndGetOutputSize("abc".AsSpan(), 0, 4));
+            Assert.Throws<FormatException>(() => Base64UrlEncoding.ValidateAndGetOutputSize("abcde".AsSpan(), 0, 5));
+
+            int result = Base64UrlEncoding.ValidateAndGetOutputSize("abc".AsSpan(), 0, 0);
+            Assert.Equal(0, result);
+
+            result = Base64UrlEncoding.ValidateAndGetOutputSize("abcd".AsSpan(), 0, 4);
+            Assert.Equal(3, result);
+
+            result = Base64UrlEncoding.ValidateAndGetOutputSize("abc=".AsSpan(), 0, 4);
+            Assert.Equal(2, result);
```

Comment:
nit: makes reading the test slightly more obvious.
```suggestion
            int actualOutputSize = Base64UrlEncoding.ValidateAndGetOutputSize("abc".AsSpan(), 0, 0);
            Assert.Equal(0, actualOutputSize);

            actualOutputSize = Base64UrlEncoding.ValidateAndGetOutputSize("abcd".AsSpan(), 0, 4);
            Assert.Equal(3, actualOutputSize);

            actualOutputSize = Base64UrlEncoding.ValidateAndGetOutputSize("abc=".AsSpan(), 0, 4);
            Assert.Equal(2, actualOutputSize);
```